pkg/config/read.go

```diff
@@ -20,6 +21,10 @@ func ReadConfig() (HarvesterConfig, error) {
 	if err != nil {
 		return *result, err
 	}
+	err = ToNetworkInterface(data)
```

Comment:
Since this hack is kernel parameter-specific, would it be better to put the logic in `parseCmdLine` function?
https://github.com/harvester/harvester-installer/blob/e0c7293a149bd385399b41766ed2fe7fc2a46006/pkg/util/cmdline.go#L13

Reply:
thanks for the feedback @bk201. Please let me know if it looks better now.